src/css/components/_c-checklist.scss

```diff
@@ -282,3 +282,11 @@ margin-left: 1rem;
 		padding-left: map-get($_checklist-inset, large);
 	}
 }
+
+.u-title-case {
+	text-transform: lowercase;
+}
```

Comment:
Could we add this to our [text-transform utility partial](https://github.com/a11yproject/a11yproject.com/blob/main/src/css/utilities/_text-transform.scss) instead?